iNaturalist/src/main/java/org/inaturalist/android/ObservationListActivity.java

```diff
@@ -747,15 +764,12 @@ public void onResume() {
         }
 
         if (mFromObsEdit) {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-                finishAndRemoveTask();
-            } else {
-                finish();
-            }
+            finishAndRemoveTask();
         }
 
 
         if (!mApp.loggedIn()) {
+            // TODO this looks like a bug
```

Comment:
You're right, please change the second `mTotalIdentifications` to `mTotalSpecies`